packages/client/src/rpc/util/CLConnectionManager.ts

```diff
@@ -248,7 +251,7 @@ export class CLConnectionManager {
     if (
       [ConnectionStatus.Disconnected, ConnectionStatus.Uncertain].includes(this.connectionStatus)
     ) {
-      this.config.logger.info('Consensus client connection established', { attentionCL: 'CL' })
+      this.config.superMsg('Consensus client connection established', { attentionCL: 'CL' })
```

Comment:
Side note: this is appearing relatively often now as a super msg, but this is rather a problem with the disconnect-detection-logic where we have some flaw or missing some connected-timestamp update (so it's not so simple to just make the threshold larger, since the current one (30 seconds or so) already does not match at all with client interaction in some occasions.

Might have a quick look here too on some follow up PR though.